fhir-bucket/src/main/java/com/ibm/fhir/bucket/app/Main.java

```diff
@@ -935,6 +975,7 @@ public void process() {
             if (this.targetBucket == null || this.targetBucket.length() == 0) {
                 // Set up the client we use to send requests to the FHIR server
                 fhirClient = new FHIRBucketClient(new ClientPropertyAdapter(fhirClientProperties));
+                fhirClient.addHeader("Prefer", "return=representation");
```

Comment:
I think this will cause the server to include the resource content in the response of each create/udpate...do we really want that?

Reply:
I see we used to add this on line 186 of FHIRBucketClient, so I guess its not really a change

Reply:
could use the new Headers.PREFER_HEADER constant

Reply:
yeah it was just a slight refactor so I wanted to keep it the same. Now using the Headers constant.